src/applications/TrainingModule.cpp

```diff
@@ -2174,7 +2174,8 @@ std::vector<int> TrainingModule::EffectSizeBasedFeatureSelection(const VariableS
     //check crossvalidated performance after adding the current feature
     double bestCV = 0;
     double bestC = 1;
-    double bestG = 1 / currentFeatureSet.Cols();
+    double numerator = 1;
```

Comment:
Why is numerator defined as ```1``` if it isn't being used anywhere else?

Reply:
removed.

Reply:
using in line 2178 of TrainingModule.cpp